homeassistant/components/synology_dsm/const.py

```diff
@@ -41,15 +41,15 @@
     f"{SynoStorage.API_KEY}:disk_exceed_bad_sector_thr": {
         ENTITY_NAME: "Exceeded Max Bad Sectors",
         ENTITY_UNIT: None,
-        ENTITY_ICON: "mdi:test-tube",
-        ENTITY_CLASS: None,
+        ENTITY_ICON: "mdi:checkbox-marked-circle-outline",
+        ENTITY_CLASS: "safety",
```

Comment:
Either set an icon or device class, not both.

Reply:
ok, l've taken out the icons.

please have a look at my comment, the temp sensors have an icon And device_class set too..

Reply:
Yes in general these should not be combined. When setting a device class, it is up to the frontend to provide for icons.

Reply:
Other sensors fixed in #39419

Reply:
> Yes in general these should not be combined. When setting a device class, it is up to the frontend to provide for icons.

cool, changed accordingly. Only Status sensor is still to be reviewed? Has a different state though: `'normal'` ... Not sure if we can set it to d_c 'Problem' ?